### PR TITLE
Chore: App 1111 - remove rinkeby

### PR DESCRIPTION
**Pull request by WissenIstNacht**
## Description

Removes all remaining references to Rinkeby.

Task: [App-1111](https://aragonassociation.atlassian.net/browse/APP-1111)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: packages/web-app/src/utils/constants/chains.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* SUPPORTED NETWORK TYPES ================================================== */
 
-const SUPPORTED_CHAIN_ID = [1, 4, 5, 137, 80001, 42161, 421611] as const;
+export const SUPPORTED_CHAIN_ID = [1, 5, 137, 80001, 42161, 421611] as const;
```
Comment (Fabricevladimir) [inline packages/web-app/src/utils/constants/chains.ts:3]: I've nothing to complain about so like this should be `SUPPORTED_CHAIN_IDS` with an 's'

File: packages/web-app/src/context/augmentedWallet.tsx [hunk 2]
```diff
@@ -76,13 +76,13 @@ export const connectors: Connector[] = [
   {
     id: 'injected',
     properties: {
-      chainId: [1, 4, 137, 80001, 42161, 421611],
+      chainId: [...SUPPORTED_CHAIN_ID],
```
Comment (Fabricevladimir) [inline packages/web-app/src/context/augmentedWallet.tsx:79]: ❤️

**Review by Fabricevladimir — APPROVED** [review]
Thanks, Mr. Roots; looking good

**Review by sepehr2github — APPROVED** [review]
Thanks

**Pull request merged**